### PR TITLE
Testing new Object Storage Page[WIP]

### PR DESCRIPTION
**Pull request by AviadP**
4.18 new feature

### CODE REVIEW

File: ocs_ci/ocs/ui/page_objects/object_buckets_tab.py
Comment (shyRozen) [inline ocs_ci/ocs/ui/page_objects/object_buckets_tab.py:1]: Is there a reason for deleting this file and re-create it with another name?

File: ocs_ci/framework/conf/default_config.yaml [hunk 1]
```diff
@@ -321,7 +321,7 @@ EXTERNAL_MODE:
 UI_SELENIUM:
   browser_type: "chrome"
   chrome_type: "google-chrome"
-  headless: True
+  headless: False
```
Comment (DanielOsypenko) [inline ocs_ci/framework/conf/default_config.yaml:324]: probably a leftover. we need to revert this change

**Comment by AviadP** [comment]
created new PR after merge of initial PR
